back/app/models/concerns/post.rb

```diff
@@ -51,6 +51,7 @@ module Post
 
     scope :draft, -> { where(publication_status: 'draft') }
     scope :published, -> { where publication_status: 'published' }
+    scope :submitted, -> { where publication_status: %w[submitted published] }
```

Comment:
Yes true, I was hesitant about this, but I thought it would make sense because we also refer to "published" as "submitted", for example in the new idea for admin: https://www.notion.so/govocal/Proposals-UX-UI-d8fd419bc1384171b13e6a06f6f9b05a?pvs=4#4bba6bd20d0145ffb505bc62f76bdbba.

But I'll change it here, as indeed it might not be clear to other devs whether or not the published status is taken into account.

Reply:
Sadly, Rubocop does not allow the name "has_been_submitted" so I chose "submitted_or_published" instead.